pkg/cloud/aws/actuators/machine/actuator_test.go

```diff
@@ -216,6 +230,15 @@ func TestActuator(t *testing.T) {
 	machineInvalidProviderConfig.Spec.ProviderConfig.Value = nil
 	machineInvalidProviderConfig.Spec.ProviderConfig.ValueFrom = nil
 
+	machineNoClusterID := machine.DeepCopy()
+	delete(machineNoClusterID.Labels, providerconfigv1.ClusterIDLabel)
+
+	pendingInstance := stubInstance("ami-a9acbbd6", "i-02fcb933c5da7085c")
+	pendingInstance.State = &ec2.InstanceState{
+		Name: aws.String(ec2.InstanceStateNamePending),
+		Code: aws.Int64(16),
```

Comment:
Create a symbolic constant to convey what 16 means.

Reply:
Removed